src/status_im2/subs/shell.cljs

```diff
@@ -74,7 +74,7 @@
   [chat id communities]
   {:title               (:chat-name chat)
    :avatar-params       {}
-   :customization-color (or (:customization-color chat) :primary)
+   :customization-color (or (:color chat) :primary)
```

Comment:
hmm, perhaps we can put :primary in a const for it's use case here. e.g default shell header color